noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -356,58 +370,556 @@
+                        COALESCE(series.val, 0) number_of_series_deletions_apart_from_disposals,
+                        COALESCE(document.val, 0) number_of_document_descriptions_deletions_apart_from_disposals
+                    FROM arkivstruktur.arkivdel s
+                    LEFT JOIN
                     (
-                        SELECT LOWER(value) value
-                        FROM addml.property
-                        WHERE name = 'endringslogg.xsd'
-                    ) detected;
-                ]]>
-            </info>
-            <warnings>
-                <![CDATA[
-                    SELECT 'endringslogg.xsd' file_name, detected.value detected_checksum,
-                        (
-                            SELECT LOWER(property.value) value
-                            FROM addml.property property
-                            JOIN addml.properties parent_properties ON property._parent_id = parent_properties._id
-                            JOIN addml.property parent_property ON parent_properties._parent_id = parent_property._id
-                            JOIN addml.property sibling_property ON sibling_property._parent_id = parent_property._parent_id
-                            WHERE property.name = 'value' AND parent_property.name = 'checksum' AND sibling_property.name = 'name' AND LOWER(sibling_property.value) = 'endringslogg.xsd'
-                            LIMIT 1
-                        ) arkivuttrekk_checksum
-                    FROM
+                        SELECT sl._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.sletting sl
+                        GROUP BY sl._ref_arkivdel
+                    ) series ON series.series_system_id = s.systemid
+                    LEFT JOIN
                     (
-                        SELECT LOWER(value) value
-                        FROM addml.property
-                        WHERE name = 'endringslogg.xsd'
-                    ) detected
-                    WHERE detected.value <> arkivuttrekk_checksum OR arkivuttrekk_checksum IS NULL OR detected.value IS NULL;
+                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.sletting sl
+                        JOIN arkivstruktur.dokumentbeskrivelse d ON d.systemid = sl._ref_dokumentbeskrivelse
+                        JOIN arkivstruktur.registrering r ON r.systemid = d._ref_registrering
+                        GROUP BY r._ref_arkivdel
+                    ) document ON document.series_system_id = s.systemid;
                 ]]>
-            </warnings>
+            </info>
         </queries>
-    </rule>
+    </check>
 
-    <rule>
+    <!-- Tests -->
+
+    <test id="AST1">
         <title>Extraction-wide systemid field uniqueness</title>
+        <description>Tests whether the same system ID has been used for more than one entity in the archive</description>
```

Comment:
`Tests whether the same systemID has been used for more than one object in the archive`